apps/laboratory/src/components/Solana/SolanaSendTransactionTest.tsx

```diff
@@ -6,16 +6,15 @@ import {
   Transaction,
   TransactionMessage,
   VersionedTransaction,
-  SystemProgram,
-  Connection
+  SystemProgram
 } from '@solana/web3.js'
 
 import { solana } from '../../utils/ChainsUtil'
 import { useChakraToast } from '../Toast'
 
 const PHANTOM_TESTNET_ADDRESS = '8vCyX7oB6Pc3pbWMGYYZF5pbSnAdQ7Gyr32JqxqCy8ZR'
 const recipientAddress = new PublicKey(PHANTOM_TESTNET_ADDRESS)
-const amountInLamports = 100000000
+const amountInLamports = 10000000
```

Comment:
Can we use underscores for better readability?

```suggestion
const amountInLamports = 10_000_000
```